gensrc/script/functions.py

```diff
@@ -630,6 +630,8 @@
     [91000, 'sub_bitmap', 'BITMAP', ['BITMAP', 'BIGINT', 'BIGINT'], 'BitmapFunctions::sub_bitmap', False],
     [91001, 'bitmap_subset_limit', 'BITMAP', ['BITMAP', 'BIGINT', 'BIGINT'], 'BitmapFunctions::bitmap_subset_limit', False],
     [91002, 'bitmap_subset_in_range', 'BITMAP', ['BITMAP', 'BIGINT', 'BIGINT'], 'BitmapFunctions::bitmap_subset_in_range', False],
+    [91003, 'bitmap_to_binary', 'VARCHAR', ['BITMAP'], 'BitmapFunctions::bitmap_to_binary', False],
```

Comment:
why not return `varbinary` type directly?

Reply:
Good suggestion, I'll think about it again

Reply:
Fixed